demos/src/hover.mustache

```diff
@@ -0,0 +1,17 @@
+<div class="o-colors-page-background">
+	<div class='demo-tooltip-container' id="box-tooltip">
+
+		<button class='o-buttons o-buttons--big demo-tooltip-target' id="demo-tooltip-target">
+			Share
+		</div>
+
+		<div data-o-component="o-tooltip"
+		data-o-tooltip-position="right"
+		data-o-tooltip-target="demo-tooltip-target"
+		data-o-tooltip-show-on-hover="true">
```

Comment:
@JakeChampion AFAIK, `Element.getAttribute()` will always return a string (or `null`). I'd updated all the attributes in the documentation to reflect this fact (and the fact that quoting attributes is a good practice given that things like unquoted `mailto:` strings will break things. See [mothereff.in/unquoted-attributes](https://mothereff.in/unquoted-attributes) and [MDN: `Element.getAttribute()`](https://developer.mozilla.org/en/docs/Web/API/Element/getAttribute).